operations/iuf/workflows/configuration.md

```diff
@@ -92,13 +92,13 @@ required for initial installation scenarios.
 
 - USS
     - Compute Configuration
-        - Configure DVS and LNet with appropriate Slingshot settings
-        - Configure DVS and LNet for use on application nodes
+        - Configure LNet with appropriate Slingshot settings
+        - Configure LNet for use on application nodes
```

Comment:
Since DVS is still used for non-CSM scenarios, the above 2 lines (and one a little below this) technically don't need to be changed, not that this change is necessarily harmful.